scripts/install

```diff
@@ -49,6 +49,9 @@ ynh_app_setting_set --app=$app --key=path --value=$path_url
 ynh_app_setting_set --app=$app --key=admin --value=$admin
 ynh_app_setting_set --app=$app --key=user_home --value=$user_home
 
+maintenance_mode=0
+ynh_app_setting_set --app=$app --key=maintenance_mode --value=$maintenance_mode
```

Comment:
Hmmm why do you create this setting? I don't remember, is it mandatory to be able to use it in the config panel?

Reply:
I'm almost offline in the montains now, so I wont be able to test. I think I got an error that the parameter was not initialized when loading the config panel.

I'll let you try it and fix it. :)

Reply:
> is it mandatory to be able to use it in the config panel?

Yes, it is :/